skunkworks_crow/src/main/java/org/odk/share/views/ui/settings/CustomEditTextPreference.java

```diff
@@ -0,0 +1,73 @@
+package org.odk.share.views.ui.settings;
```

Comment:
We should use some names that can indicate the class usage, not just `CustomEditTextPreference` @Chromicle .

Reply:
I gave that name because maybe it will be useful for further implementations of any features